e2e/docker_compose_redpanda.yaml

```diff
@@ -3,6 +3,7 @@ version: "3"
 services:
   # is redpanda: reusing names to not break kafka integration tests
   kafka:
+    image: docker.redpanda.com/redpandadata/redpanda:v23.3.1
```

Comment:
nit: can we also use the env var versioning style here?

Reply:
Sure.